docs/mobile/exercises/README.md

```diff
@@ -3,14 +3,14 @@
 There are roughly 5 exercise types available to you as an author on DataCamp
 Mobile:
 
-| Type                                  | Description                                             |
-| :---                                  | :----------                                             |
-| [Select Output](select-output.md)     | What is the output of this code? Console output         |
-| [Select Table](select-table.md)       | What is the output of this code? Formatted table output |
-| [Select Code](select-code.md)         | Select the code to return the output.                   |
-| [Tap](tap.md)                         | Complete the code to return the output.                 |
-| [Multiple Choice](multiple-choice.md) | Answer the multiple choice question.                    |
-| [Reorder](reorder.md)                 | Reorder the code to return the output.                  |
+| Type                                                    | Description                                             |
+| :---                                                    | :----------                                             |
+| [Select Output](/mobile/exercises/select-output.md)     | What is the output of this code? Console output         |
+| [Select Table](/mobile/exercises/select-table.md)       | What is the output of this code? Formatted table output |
+| [Select Code](/mobile/exercises/select-code.md)         | Select the code to return the output.                   |
+| [Tap](/mobile/exercises/tap.md)                         | Complete the code to return the output.                 |
+| [Multiple Choice](/mobile/exercises/multiple-choice.md) | Answer the multiple choice question.                    |
+| [Reorder](/mobile/exercises/reorder.md)                 | Reorder the code to return the output.                  |
 
 ...but they all share a similar structure. In general, you can think of an
```

Comment:
I would add a screenshot for the exercise below, so people can get an idea at first glance about context, question, code...

Reply:
right